starter-code/views/layouts/main.ejs

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <form action="POST" action="/charts/bitcoin">
+        <label for="start"> From </label>
+        <input type="date" id="start" placeholder="01.12.2017" ></input>
+        <label for="end"> to </label>
+        <input type="date" id="end" placeholder="31.12.2017" ></input>
+        <button type="submit" id="submitButton">Go</button>
+    </form>
```

Comment:
It's not necessary do a `POST` request since you're running your script on front-end. What you can do is put a `eventListenner` on de inputs every change it changes or in the button `submitButton` every time you click it.